RoddyCore/src/de/dkfz/roddy/knowledge/files/BaseFile.groovy

```diff
@@ -345,7 +345,11 @@ abstract class BaseFile<FS extends FileStageSettings> extends FileObject {
 
         if (helper instanceof ConstructionHelperForGenericCreation) { // Manual creation is currently intrinsic.
             ConstructionHelperForGenericCreation _helper = helper as ConstructionHelperForGenericCreation
-            if (_helper.parentObject instanceof FileGroup) {
+            def _pFiles = helper.parentFiles
```

Comment:
`_pFiles` -> `parents`

This "_" notation is hardly ever used -- even in our code. pFiles is also much less understandible.

Reply:
Removed the temporary variable. Only used twice.